src/multisig/multisig.safe.constants.ts

```diff
@@ -0,0 +1,19 @@
+export enum Network {
+    Mainnet = 1,
+    Sepolia = 11155111,
+    BSC = 56,
+    Polygon = 137,
+    Arbitrum = 42161,
+    Optimism = 10,
```

Comment:
We don't support `Arbitrum` and `Optimism` at the moment, so even if the user has a Gnosis Safe Multi-Sig - let's keep these out of our list _(worse case they can choose `Other`)_.